src/Uno.UI.Wasm/ts/WindowManager.ts

```diff
@@ -1589,6 +1590,10 @@
 				if (!WindowManager.dispatchEventMethod) {
 					WindowManager.dispatchEventMethod = (<any>Module).mono_bind_static_method("[Uno.UI] Windows.UI.Xaml.UIElement:DispatchEvent");
 				}
+
+				if (!WindowManager.focusInMethod) {
+					WindowManager.focusInMethod = (<any>Module).mono_bind_static_method("[Uno.UI] Windows.UI.Xaml.Input.FocusManager:ReceiveFocusNative");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5251853)